### ISSUE TITLE
[FEATURE] Don't force video content to be entered in video player when not fullscreen

### ISSUE THREAD

**Issue by frogman1189** [jhomlala/betterplayer#633]
**Is your feature request related to a problem? Please describe.**
When placing BetterPlayer inside a ConstrainedBox, and BetterPlayer is not full screen, the better player expands to fill the box, and then places the actual video entered within that box rather than taking the minimum constraints.

**Describe the solution you'd like**
This is caused by having wrapped the content player in a center in the below code
```
Center(
      child: Container(
        width: double.infinity,
        color: betterPlayerController
            .betterPlayerConfiguration.controlsConfiguration.backgroundColor,
        child: AspectRatio(
          aspectRatio: aspectRatio,
          child: _buildPlayerWithControls(betterPlayerController, context),
        ),
      ),
    );
```
in the `_BetterPlayerWithControlsState.build()` function. I would appreciate it if there was a check to see whether the player is currently fullscreen, and if not leave it to the user to align it how they wish

**Describe alternatives you've considered**
The alternative is to wrap the entire player inside yet another AspectRatio and have precomputed the aspect ratio

**Additional context**
Expected:
![Screen Shot 2021-08-03 at 11 58 25 AM](https://user-images.githubusercontent.com/20494957/127937752-9f0deb49-b67d-4b37-aac7-78cf2d3328aa.png)

Real:
![Screen Shot 2021-08-03 at 11 58 03 AM](https://user-images.githubusercontent.com/20494957/127937766-fe6dafdb-2ea9-4b32-89f2-2e0cf7243151.png)

**Comment by jhomlala** [comment]
@frogman1189 Thanks for reporting issue, can you provide minimal reproducible example? This will help me debug this problem.

**Comment by frogman1189** [comment]
```
import 'package:better_player/better_player.dart';
import 'package:flutter/material.dart';

void main() {
  runApp(const MyApp());
}

class MyApp extends StatelessWidget {
  const MyApp({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return MaterialApp(
      home: Scaffold(
        body: Center(
          child: Column(
            mainAxisSize: MainAxisSize.min,
            children: [
              Player(),
              Player(),
            ],
          ),
        ),
      ),
    );
  }
}

class Player extends StatelessWidget {
  const Player({Key? key}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return ConstrainedBox(
      constraints: BoxConstraints(maxWidth: 300, maxHeight: 300),
      child: Card(
        elevation: 4,
        color: Colors.green,
        child: BetterPlayer.network(
          'https://sample-videos.com/video123/mp4/720/big_buck_bunny_720p_1mb.mp4',
        ),
      ),
    );
  }
}

```

**Comment by jhomlala** [comment]
Thank you for great issue description and example. This will be fixed in next release.

**Comment by jhomlala** [comment]
Fixed in 0.0.74.
